OpenOversight/app/commands.py

```diff
@@ -230,7 +230,7 @@ def update_officer_field(officer_field_name):
                     new_value = parse(row[field_name])
                     if isinstance(old_value, date):
                         new_value = new_value.date()
-                except Exception as e:
+                except ValueError as e:
```

Comment:
Do we know how was this subclass was chosen?

Reply:
I chose it through a manual lookup.

Reply:
I just defaulted back to `Exception` to be safe.